tests/connection/test_nibegw_message_parsing.py

```diff
@@ -159,5 +166,48 @@ def test_parse_read_request(self):
         self.assertEqual(binascii.hexlify(raw), b"c06b06393006120f00bf")
 
 
+class MessageGenerigRequestParsingTestCase(unittest.TestCase):
+
+    @staticmethod
+    def _parse_hexlified_raw_message(txt_raw):
+        raw = binascii.unhexlify(txt_raw)
+        data = GenericRequest.parse(raw)
+        value = data.fields.value
+        print(value)
+        return value
+
+    def test_parse_write_request(self):
+        hex = bytes([192,107,6,115,176,1,0,0,0,111]).hex()
```

Comment:
I see you do not like hex =)

Reply:
Heh, copied from nibepi. Was just easier.